ui/ozone/platform/wayland/wayland_pointer.h

```diff
@@ -57,12 +57,17 @@ class WaylandPointer {
 
   void SetSerial(uint32_t serial);
 
+  int GetFlagsWithKeyboardModifiers();
+
   WaylandConnection* connection_ = nullptr;
   std::unique_ptr<WaylandCursor> cursor_;
   wl::Object<wl_pointer> obj_;
   EventDispatchCallback callback_;
   gfx::PointF location_;
   int flags_ = 0;
+
+  // Keeps track of current modifiers.
+  int modifiers_ = 0;
```

Comment:
do we need modifiers_ at all? instead, maybe we can always just call connection_->GetKeyboardModifiers?

if we ultimately need it, then rename it to keyboard_modifiers_ ?

Reply:
We keep track of old modifiers in order to remove them before updating current flags with new modifiers. Otherwise, you never know if one must remove or add additional flags(modifiers) to current flags.